### PR TITLE
Pylint fixes

### PR DESCRIPTION
**Pull request by bentsi**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Review by juliayakovlev — DISMISSED** [review]

File: sdcm/ec2_client.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def _get_ec2_client(self, region_name=None) -> EC2Client:
             return boto3.client(service_name='ec2', region_name=region_name)
         except NoRegionError:
```
Comment (dkropachev) [inline sdcm/ec2_client.py:57]:
```
        except NoRegionError as error:
            if not region_name:
                raise CreateEC2ClientNoRegionError() from error
```

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -97,11 +97,11 @@ class NoMandatoryParameter(Exception):
     """ raised from within a nemesis execution to skip this nemesis"""
 
 
-class DefaultValue:
+class DefaultValue:  # pylint: disable=too-few-public-methods
     """
     This is class is intended to be used as default value for the cases when None is not applicable
     """
-    pass
+    ...
```
Comment (dkropachev) [inline sdcm/nemesis.py:104]: Having docstring allows you have nothing in the body

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: sdcm/nemesis.py (hunk 2 above)
Comment (bentsi) [reply sdcm/nemesis.py:104]: Yep

File: sdcm/ec2_client.py (hunk 1 above)
Comment (bentsi) [reply sdcm/ec2_client.py:57]: ooops

Comment (bentsi) [reply sdcm/ec2_client.py]: fixed

**Review by juliayakovlev — APPROVED** [review]

File: sdcm/microbenchmarking.py [hunk 3]
```diff
@@ -481,9 +481,10 @@ def exclude_testrun_by_commit_id(self, commit_id=None):
             return
 
         for doc in results['hits']['hits']:
-            self.log.info("Exlcude test: {}\nCommit: #{}\nRun Date time: {}\n".format(doc['_id'],
-                                                                                      doc['_source']['versions']['scylla-server']['commit_id'],
-                                                                                      doc['_source']['test_run_date']))
+            self.log.info("Exlcude test: %s\nCommit: #%s\nRun Date time: %s\n",
```
[the remark this answers is not in the document]
Comment (bentsi) [reply sdcm/microbenchmarking.py]: fixed

File: ics_space_amplification_goal_test.py [hunk 4]
```diff
@@ -132,18 +131,18 @@ def _get_nodes_space_ampl_over_time_gb(self, dict_nodes_initial_capacity, start_
...
+            self.log.info("Space amplification is: %s GB", dict_nodes_space_amplification[node.private_ip_address])
         return dict_nodes_space_amplification
 
     def measure_nodes_space_amplification_after_write(self, dict_nodes_initial_capacity, written_data_size_gb,
                                                       start_time):
-        self.log.info(f"Space amplification results after a write of: {written_data_size_gb} are:")
         dict_nodes_space_amplification = self._get_nodes_space_ampl_over_time_gb(
             dict_nodes_initial_capacity=dict_nodes_initial_capacity,
             written_data_size_gb=written_data_size_gb, start_time=start_time)
+        self.log.info("Space amplification results after a write of: %s are: %s",
```
[the remark this answers is not in the document]
Comment (bentsi) [reply ics_space_amplification_goal_test.py:144]: let's keep it.

**Pull request merged**